www-apps/redmine/redmine-4.1.1.ebuild

```diff
@@ -0,0 +1,229 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby25 ruby26"
+inherit eutils depend.apache ruby-ng
+
+DESCRIPTION="Flexible project management web application using the Ruby on Rails framework"
+HOMEPAGE="https://www.redmine.org/"
+SRC_URI="https://www.redmine.org/releases/${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="imagemagick fastcgi ldap markdown mysql passenger postgres sqlite"
+
+DEPS="
+	fastcgi? ( dev-ruby/fcgi )
+	imagemagick? ( dev-ruby/mini_magick )
+	ldap? ( dev-ruby/ruby-net-ldap )
+	markdown? ( >=dev-ruby/redcarpet-3.5.0 )
+	mysql? ( >=dev-ruby/mysql2-0.5.0:0.5 )
+	postgres? ( >=dev-ruby/pg-1.1.4:1 )
+	sqlite? ( >=dev-ruby/sqlite3-1.4.0 )
+	dev-ruby/actionpack-xml_parser:2
+	>=dev-ruby/i18n-1.6.0:1
+	>=dev-ruby/mail-2.7.1
+	dev-ruby/mimemagic
+	>=dev-ruby/mini_mime-1.0.1
+	>=dev-ruby/nokogiri-1.10.0
+	dev-ruby/rails:5.2
+	>=dev-ruby/rbpdf-1.20.0
+	dev-ruby/request_store:0
+	>=dev-ruby/roadie-rails-2.1.0
+	>=dev-ruby/rouge-3.12.0
+	>=dev-ruby/ruby-openid-2.9.2
+	dev-ruby/rack-openid
+"
+
+ruby_add_rdepend "${DEPS}"
+ruby_add_bdepend "${DEPS}"
```

Comment:
Why are these deps also specified as build deps? If you need them for tests then they should be conditional on the test use flag, but the ruby eclasses already take care of this automatically.